wcfsetup/install/lang/de.xml

```diff
@@ -1505,6 +1505,8 @@ Als Benachrichtigungs-URL in der Konfiguration der sofortigen Zahlungsbestätigu
 		<item name="wcf.acp.option.message_sidebar_enable_articles"><![CDATA[Artikelanzahl der Autoren anzeigen]]></item>
 		<item name="wcf.acp.option.log_missing_language_items"><![CDATA[Fehlende Sprachvariablen protokollieren]]></item>
 		<item name="wcf.acp.option.log_missing_language_items.description"><![CDATA[Die fehlenden Sprachvariablen werden in der Datei <code>{'WCF_DIR'|constant}log/missingLanguageItems.txt</code> protokolliert.]]></item>
+		<item name="wcf.acp.option.enable_deprecation_warnings"><![CDATA[Fehler bei Verwendung von @deprecated code]]></item>
+		<item name="wcf.acp.option.enable_deprecation_warnings.description"><![CDATA[Verhindert die Verwendung von @deprecated Klassen und Methoden, indem bei Verwendung ein <kbd>\Error</kbd> geworfen wird.]]></item>
```

Comment:
`Klassen und Methoden` might be too restrictive as it can also be relevant for parts of methods which would then also not rely on `@deprecated` (thus, simply replace with `Code`?). On the other hand, may-be this objection it is too specific.

Reply:
I think that the specific examples are helpful to understand what exactly happens here. “Preventing using code” is somewhat vague.

Reply:
It would `Verhindert die Verwendung von @deprecated Code`.